eng/common/scripts/Verify-Readme.ps1

```diff
@@ -1,15 +1,23 @@
 # Wrapper Script for Readme Verification
 [CmdletBinding()]
 param (
-  [string]$DocWardenVersion = "0.7.1",
-
+  [string]$DocWardenVersion = "0.7.2",
```

Comment:
I assume we should either just default the version either here or in the yml but not both.